changelog/fix17630.dd

```diff
@@ -0,0 +1,5 @@
+Fix Issue 17630 - selective imports find symbols in private imports of other modules
+
+A private module-level import should not have any symbols
+visible outside the module scope. For more information
+see https://issues.dlang.org/show_bug.cgi?id=17630
```

Comment:
I prefer that the changelog entry is extended to explain the issue and the solution inline without having to read the issue.